hpx/components/containers/partitioned_vector/detail/make_index_sequence.hpp

```diff
@@ -0,0 +1,113 @@
+//  Copyright (c) 2017 Antoine Tran Tan
```

Comment:
Wouldn't it be better then to add the unrolled version to `pack.hpp` instead, so the whole library profits from it?